src/components/HarvestingConfiguration/Fields/HarvestingStatusSelect.js

```diff
@@ -7,12 +8,12 @@ import { required } from '../../../util/validate';
 import statusOptions from '../../../util/data/statusOptions';
 import useTranslateLabels from '../../../util/hooks/useTranslateLabels';
 
-const HarvestingStatusSelect = () => {
+const HarvestingStatusSelect = ({ isProviderStatusInactive }) => {
   return (
     <Field
       component={Select}
       dataOptions={useTranslateLabels(statusOptions)}
-      defaultValue="active"
```

Comment:
Is the default value removed intentionally?
The linked issue description states:
> The default values for the provider status and the harvesting status are both Active when you create a usage data provider (UDP).

Reply:
Yes, I thought it is not needed, because it is already set via `initialValues`:
`initialValues={{ harvestingConfig: { harvestingStatus: 'active', harvestVia: 'sushi', reportRelease: '5', sushiConfig: { serviceType: 'cs50' } } }}`
https://github.com/folio-org/ui-erm-usage/blob/e891a5992ba7fa5392aeb634ae9689a1eba79f64/src/routes/UDPCreateRoute.js#L54C37-L54C153
But `status: 'active'` is missing. I would add `status: 'active'` to `initialValues`